modules/dnn/src/op_webnn.hpp

```diff
@@ -40,6 +40,14 @@ class WebnnBackendNode;
 class WebnnBackendWrapper;
 
 
+template<typename T>
+inline std::vector<T> getShape(const Mat& mat)
```

Comment:
Should you add namespace for webnn related methods, say `webnn::getShape`? BTW, webnn defines `dimensions` as `vector<int32_t>`. So it would be better to change this method signature to
```c++
std::vector<int32_t> getDimensions(const Mat& mat);
```

Reply:
Thanks! I have pushed a new commit to improve this.